site/src/css/layout/_announcement.scss

```diff
@@ -0,0 +1,16 @@
+/* Announcement Bar CSS */
```

Comment:
We probably dont even need this any more since we dont use the announcement bar

Reply:
I figured we could leave it in case we need the announcement bar again so we don't have to redo that css, but if we don't want to keep it we don't have to

Reply:
I imagine we would give it new css. it doesnt really matter - just feels icky to leave unused stuff in the project